debug/default/index.ts

```diff
@@ -0,0 +1,75 @@
+import {
+	AreaSeries,
+	CandlestickSeries,
+	createChart,
+} from "lightweight-charts";
+
+const chartOptions = {
+	layout: {
+		textColor: "black",
+		background: {
+			type: "solid",
+			color: "white",
+		},
+	},
+};
```

Comment:
I got a typescript warning when I opened the file in my playground.
Lets add some types to make TS happy.
```suggestion
import {
	AreaSeries,
	CandlestickSeries,
	ChartOptions,
	ColorType,
	createChart,
	DeepPartial,
} from "lightweight-charts";

const chartOptions = {
	layout: {
		textColor: "black",
		background: {
			type: ColorType.Solid,
			color: "white",
		},
	},
} satisfies DeepPartial<ChartOptions>;
```